tests/study_tests/test_study.py

```diff
@@ -148,8 +148,19 @@ def test_optimize_with_direction() -> None:
     assert study.direction == StudyDirection.MAXIMIZE
     check_study(study)
 
-    with pytest.raises(ValueError):
+    with pytest.raises(ValueError) as e:
         create_study(direction="test")
+    assert str(e.value) == (
+        "Please set either 'minimize' or 'maximize' to direction. You can also set the "
+        + "corresponding `StudyDirection` member."
+    )
```

Comment:
IMO, we should not test the exact contents of the error message since it is not a specification, so might be changed later. It would be great to confirm that the error is emitted.